edith-step/src/App.js

```diff
@@ -1,15 +1,31 @@
 import React, {Component} from 'react';
-import ReceiptInput from './ReceiptInput';
-// import "./App.css";
 
+import ReceiptInput from './components/ReceiptInput';
+import TopNavbar from './components/TopNavbar';
 
-/** Main webpage for the website. */
+import './App.css';
+
+/**
+ * Main webpage for the website.
+ */
 class App extends Component {
-  /** Renders the main webpage. */
+
   render() {
-    /** Returns main components. */  
     return (
       <div className='App'>
+        <div className='App-header'>
+          <TopNavbar />
+        </div>
+        <div className='background-image-0'>
+          <div className='scroll-down border'>
+            Scroll Down
+          </div>
+          <div className='app-describe'>
+            <span className='border'>
+              Welcome To Edith: The Best Expenditure Analyzer.
+            </span>
+          </div>
+        </div>
         <ReceiptInput />
```

Comment:
Should probably move this to its own view and implement some kind of React navigation in a future PR.

No action necessary.